new-log-viewer/src/utils/config.ts

```diff
@@ -111,6 +104,7 @@ const getConfig = <T extends CONFIG_KEY>(key: T): ConfigMap[T] => {
 
     // Read values from `localStorage`.
     switch (key) {
+        // Do NOT handle `CONFIG_KEY.THEME` here since is handled by `CssVarsProvider`.
```

Comment:
(ditto) Can we leave the `case CONFIG_KEY.THEME:` condition? That way IDEs / linters would not complain about any missing handling for that specific case. e.g.,

```
case CONFIG_KEY.THEME:
    // Handled by JoyUI instead.
    break;
```